### PR TITLE
Contributors for Chapel 1.27

### PR DESCRIPTION
**Pull request by bradcray**
Refresh the contributors lists for Chapel 1.27.  Thanks to all who contributed to
this release!

### CODE REVIEW

**Comment by bradcray** [comment]
@ghbrown, @jeremiah-corrado, @brandon-neth  : Would you make sure I spelled and alphabetized your names correctly?

@chapel-lang/core-contributors : If you've merged PRs from external contributors in this release, would you make sure I've got them in the list?

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -6,37 +6,35 @@ most recent Chapel release:
...
 * Lydia Duncan, [HPE]
 * Michael Ferguson, [HPE]
+* Ben Harshbarger, [HPE]
 * John Hartman, [HPE]
 * David Iten, [HPE]
 * Engin Kayraklioglu, [HPE]
+* Shreyas Khandekar, [HPE] intern from [University of Arizona]
 * Vassily Litvinov, [HPE]
 * David Longnecker, [HPE]
 * Ben McDonald, [HPE] (former intern from [Gonzaga University])
```
Comment (e-kayrakli) [inline CONTRIBUTORS.md:24]: * Josh Milthorpe, [Oak Ridge National Laboratory]

**Comment by bradcray** [comment]
@milthorpe , @lucaferranti : Sorry to have missed you on my first draft.  For some reason, https://github.com/chapel-lang/chapel/graphs/contributors?from=2022-03-11&to=2022-06-17&type=c doesn't seem to list everyone who committed code... (?).  Would you check and see that I spelled and alphabetized your names correctly?

File: CONTRIBUTORS.md [hunk 2]
```diff
@@ -6,37 +6,37 @@ most recent Chapel release:
 
 Contributors to the Chapel 1.26.0 release
```
Comment (ghbrown) [inline CONTRIBUTORS.md:7]: Should this be 1.27.0?

**Comment by lucaferranti** [comment]
@bradcray yes correctly spelled and alphabetized. Glad I could help 🎉

File: CONTRIBUTORS.md (hunk 2 above)
Comment (bradcray) [reply CONTRIBUTORS.md]: Yes, thanks for catching that!

**Pull request merged**